plugins/woocommerce-blocks/tests/e2e/tests/cart/cart-checkout-block-translations.shopper.block_theme.side_effects.spec.ts

```diff
@@ -42,7 +42,15 @@ test.describe( 'Shopper → Translations', () => {
 		page,
 	} ) => {
 		await frontendUtils.goToShop();
-		await page.getByLabel( 'Toevoegen aan winkelwagen: “Beanie“' ).click();
+
+		const beanieAddToCartButton = page.getByLabel(
+			'Toevoegen aan winkelwagen: “Beanie“'
+		);
+		await beanieAddToCartButton.click();
+
+		// Add to cart initiates a request that could be interrupted by navigation, wait till it's done.
+		await expect( beanieAddToCartButton ).toHaveText( /in winkelwagen/ );
```

Comment:
These tests had a race condition that was exposed, I guess by slight changes in the bundling. If you navigate before the request to add to cart has finished you might interrupt it, so this waits for that to ensure these pass.